anta/tests/security.py

```diff
@@ -268,3 +273,106 @@ def test(self) -> None:
             self.result.is_failure(f"eAPI IPv6 ACL(s) not configured or active in vrf {self.inputs.vrf}: {not_configured_acl_list}")
         else:
             self.result.is_success()
+
+
+class VerifyAPISSLCertificate(AntaTest):
+    """
+    Verifies the eAPI SSL certificate expiry, common subject name, encryption algorithm and key size.
+
+    Expected Results:
+        * success: The test will pass if the certificate's expiry date is greater than the threshold,
+                   and the certificate has the correct name, encryption algorithm, and key size.
+        * failure: The test will fail if the certificate is expired or is going to expire,
+                   or if the certificate has an incorrect name, encryption algorithm, or key size.
+    """
+
+    name = "VerifyAPISSLCertificate"
+    description = "Verifies the eAPI SSL certificate expiry, common subject name, encryption algorithm and key size."
+    categories = ["security"]
+    commands = [AntaCommand(command="show management security ssl certificate"), AntaCommand(command="show clock")]
+
+    class Input(AntaTest.Input):
+        """
+        Input parameters for the VerifyAPISSLCertificate test.
+        """
+
+        certificates: list[APISSLCertificates]
```

Comment:
```suggestion
        certificates: List[APISSLCertificates]
```
I think CI is failing because of that. Please confirm